app/controllers/public_health_controller.rb

```diff
@@ -79,4 +80,14 @@ def authenticate_user_role
     # Restrict access to public health only
     redirect_to(root_url) && return unless current_user.can_view_public_health_dashboard?
   end
+
+  def set_jurisdiction_paths
+    @possible_jurisdiction_paths = current_user.jurisdiction.subtree.pluck(:id, :path).to_h
+  end
+
+  def set_all_assigned_users
+    # Get all assigned users of current user's jurisdiction
+    patients = current_user.patients&.where(jurisdiction_id: current_user.jurisdiction.subtree_ids)
+    @all_assigned_users = patients.where.not(assigned_user: nil).distinct.pluck(:assigned_user).sort
```

Comment:
Is the `distinct` needed?

Reply:
I got the `distinct` from here (https://github.com/SaraAlert/SaraAlert/blob/1.36.0/app/controllers/jurisdictions_controller.rb#L34). I will update this line to what you suggested.